CMakeLists.txt

```diff
@@ -119,6 +120,11 @@ else (USE_EXT_GD)
 		FIND_PACKAGE(WEBP REQUIRED)
 	ENDIF (ENABLE_WEBP)
 
+	IF (ENABLE_AVIF)
+	  FIND_PACKAGE(libavif 0.8.2 REQUIRED CONFIG)
```

Comment:
only use tabs to indent

Reply:
sorry, I keep grepping for spaces and fixing them... and then I make a change and don't do the grep :-/